### DISCUSSION TITLE
I would love to see a way to make filters from requests in f.ex Overseerr

### DISCUSSION THREAD

**Discussion by SSamiK** [autobrr/autobrr#1382] [Ideas]
It would be nice to have a way to pull requests added in software like Overseerr/Jellyseerr to generate filters in autobrr. 
I have no idea if this would be possible, but it sure would be a nice feature.

**Comment by zze0s** [comment]
Hi! It's already sort of supported, but not directly. Those services use Radarr and Sonarr and autobrr can communicate with those directly. We then have https://github.com/autobrr/omegabrr which pulls titles from the arrs and makes filters for you that only contains monitored titles.

**Reply by SSamiK** [reply]
Oh, great. I will have to look into that then. I was hoping to avoid radarr/sonarr, but I will consider adding them anyway.
